CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#568](https://github.com/rubocop/rubocop-rails/pull/568): Add new `Rails/SchemaComment` cop. ([@vitormd][])
```

Comment:
Can you use the following changelog entry format style instead this, and squash your commits into one?

> Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop-rails/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

Reply:
Done